open-api/rest-catalog-open-api.yaml

```diff
@@ -3789,6 +4012,21 @@ components:
             EmptyResponse:
               $ref: '#/components/examples/ListNamespacesEmptyExample'
 
+    PayloadTooLargeWithPlanResponse:
+      description:
+        Payload is too large. Informs user to leverage /preplan before doing /plan
```

Comment:
Should this be more direct in the spec? `This failure indicates the payload is too large. If this is encountered a client should perform /preplan followed by a /plan`

Reply:
I take this back, we probably don't want to dictate what a client should do here. I think either `informs` as is fine or just remove it and let the error message indicate the suggestion